### PR TITLE
[Docs]fix_numpy_issue

### PR DESCRIPTION
**Pull request by Ligoml**
### PR types

Others
### PR changes

Docs
### Describe

Remove the third-party library numpy in API docs(example code)

### CODE REVIEW

File: python/paddle/tensor/linalg.py [hunk 1]
```diff
@@ -1046,12 +1045,9 @@ def dot(x, y, name=None):
...
 
         import paddle
-        import numpy as np
 
-        x_data = np.random.uniform(0.1, 1, [10]).astype(np.float32)
-        y_data = np.random.uniform(1, 3, [10]).astype(np.float32)
-        x = paddle.to_tensor(x_data)
-        y = paddle.to_tensor(y_data)
+        x=paddle.uniform(shape=[10],dtype='float32', min=0.1, max=1, seed=10, name=None)
+        y=paddle.uniform(shape=[10],dtype='float32', min=1, max=3, seed=10, name=None)
```
Comment (SigureMo) [inline python/paddle/tensor/linalg.py:1050]:
```suggestion
        x = paddle.uniform(shape=[10], dtype='float32', min=0.1, max=1, seed=10, name=None)
        y = paddle.uniform(shape=[10], dtype='float32', min=1, max=3, seed=10, name=None)
```

[doge]

File: python/paddle/tensor/linalg.py [hunk 2]
```diff
@@ -634,10 +634,9 @@ def dist(x, y, p=2, name=None):
         .. code-block:: python
 
             import paddle
-            import numpy as np
 
-            x = paddle.to_tensor(np.array([[3, 3],[3, 3]]), "float32")
-            y = paddle.to_tensor(np.array([[3, 3],[3, 1]]), "float32")
+            x = paddle.to_tensor([[3, 3],[3, 3]], "float32")
+            y = paddle.to_tensor([[3, 3],[3, 1]], "float32")
```
Comment (Liyulingyue) [inline python/paddle/tensor/linalg.py:639]: suggestion:
            x = paddle.to_tensor([[3, 3], [3, 3]], "float32")
            y = paddle.to_tensor([[3, 3], [3, 1]], "float32")

File: python/paddle/tensor/linalg.py (hunk 1 above)
Comment (Ligoml) [reply python/paddle/tensor/linalg.py:1050]: 谢谢您

File: python/paddle/tensor/linalg.py (hunk 2 above)
Comment (Liyulingyue) [inline python/paddle/tensor/linalg.py:639]:
```suggestion
            y = paddle.to_tensor([[3, 3], [3, 1]], "float32")
```

File: python/paddle/tensor/linalg.py [hunk 3]
```diff
@@ -634,10 +634,9 @@ def dist(x, y, p=2, name=None):
         .. code-block:: python
 
             import paddle
-            import numpy as np
 
-            x = paddle.to_tensor(np.array([[3, 3],[3, 3]]), "float32")
-            y = paddle.to_tensor(np.array([[3, 3],[3, 1]]), "float32")
+            x = paddle.to_tensor([[3, 3],[3, 3]], "float32")
```
Comment (Liyulingyue) [inline python/paddle/tensor/linalg.py:638]:
```suggestion
            x = paddle.to_tensor([[3, 3], [3, 3]], "float32")
```

**Pull request closed without merging**